spec/system/privacy_spec.rb

```diff
@@ -0,0 +1,177 @@
+require 'rails_helper'
+
+RSpec.describe('Private Fields') do
+  before do
+    driven_by(:rack_test)
+    @user = FactoryBot.create(:user)
+    @user2 = FactoryBot.create(:user2)

```

Comment:
user2 is fine
In the future consider
```rb
Factorybot.create(:user, email: "other@gmail.com", ...)
```

as you can override parameters in the create function.